MISampleApp/SFMISampleConsole/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM mcr.microsoft.com/windows/nanoserver:1809
```

Comment:
We should add a comment here showing what are compatible Win OS images that can support this type of container. Ideally a link to the compatibility doc (https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility)